mods/d2k/chrome/ingame-player.yaml

```diff
@@ -2,8 +2,13 @@ Container@PLAYER_WIDGETS:
 	Logic: LoadIngameChatLogic
 	Children:
 		Container@CHAT_ROOT:
-		LogicKeyListener@CONTROLGROUP_KEYHANDLER:
-			Logic: ControlGroupLogic
+		ControlGroups@CONTROLGROUPS:
+			SelectGroupKeyPrefix: ControlGroupSelect
+			CreateGroupKeyPrefix: ControlGroupCreate
+			AddToGroupKeyPrefix: ControlGroupAddTo
+			CombineWithGroupKeyPrefix: ControlGroupCombineWith
+			JumpToGroupKeyPrefix: ControlGroupJumpTo
+			HotkeyCount: 10
```

Comment:
We can remove these too, although it requires a little more faff: https://github.com/OpenRA/OpenRA/commit/b6af5479b6580c06814cb0aead6cc089bbc7e71e

Reply:
:+1: :+1: Thanks, I was looking for a way to do this.